src/resources/ApiKeys/ApiKeys.ts

```diff
@@ -31,4 +31,12 @@ export default class ApiKey extends Resource {
     delete(apiKeyId: string) {
         return this.api.delete(`${ApiKey.baseUrl}/${apiKeyId}`);
     }
+
+    extend(apiKeyId: string) {
+        return this.api.put(`${ApiKey.baseUrl}/${apiKeyId}/activation/extend`);
```

Comment:
This one returns a 204 (no content), so we can type its return as `void`

```suggestion
        return this.api.put<void>(`${ApiKey.baseUrl}/${apiKeyId}/activation/extend`);
```